NumberBaseball/NumberBaseball/Step1.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+
+var computerRandomNumbers: Set<Int> = []
+var randomNumbers: Set<Int> = []
+
+func createComputerRandomNumbers() -> [Int] {
+    while computerRandomNumbers.count != 3 {
+        computerRandomNumbers.insert(Int.random(in: 1...9))
+    }
+    let computerRandomNumbers = Array(computerRandomNumbers)
+    return computerRandomNumbers
+}
+
+func createRandomNumbers() -> [Int] {
+    randomNumbers.removeAll()
+    while randomNumbers.count != 3 {
+        randomNumbers.insert(Int.random(in: 1...9))
+    }
+    let randomNumbers = Array(randomNumbers)
+    return randomNumbers
+}
+
+func judgeResult(computerRandomNumbersArray: [Int],randomNumbersArray: [Int]) -> [Int] {
+    var ball: Int = 0
+    var strike: Int = 0
+    var resultOfStrikeAndBall: [Int] = []
+    
+    for i in 0...2 {
+        if computerRandomNumbersArray[i] == randomNumbersArray[i] {
+            strike += 1
+        } else if computerRandomNumbersArray[i] != randomNumbersArray[i] && randomNumbersArray.contains(computerRandomNumbersArray[i]) {
+            ball += 1
+        }
+    }
+    resultOfStrikeAndBall.append(strike)
+    resultOfStrikeAndBall.append(ball)
+    return resultOfStrikeAndBall
+}
+
+func startGame() {
+    var remainCount: Int = 9
+    let computerRandomNumbersArray = createComputerRandomNumbers()
+    let randomNumbersArray = createRandomNumbers()
+    let judgeResult = judgeResult(computerRandomNumbersArray: computerRandomNumbersArray, randomNumbersArray: randomNumbersArray)
+    
+    while remainCount != 0 && judgeResult[0] != 3 {
+        print("임의의 수 : \(randomNumbersArray[0]) \(randomNumbersArray[1]) \(randomNumbersArray[2])")
+        print("\(judgeResult[0]) 스트라이크, \(judgeResult[1]) 볼")
```

Comment:
코드가 짧다고 무조건 좋은 코드가 아닙니다
개발한 사람이 아니라면 해당 키워드가 무엇을 의미하는 것인지 모를 수도 있어요

```suggestion
     let strike = judgeResult[0]
     let ball = judgeResult[1]
     print("\(strike) 스트라이크, \(ball) 볼")
```

지역변수를 활용해 명시적으로 보여주는 것도 하나의 방법입니다

Reply:
튜플을 사용해서 묶어주고 튜플유형의 요소에 이름을 지정하고 해당이름을 사용하여 개별요소의 값을 창조할 수 있게 하였습니다.
가독성측면에서도 더 나은 부분이 무엇인지 고민하였습니다!